src/formatDate.js

```diff
@@ -50,7 +50,74 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let year;
+  let month;
+  let day;
+  let separator = fromFormat[3];
```

Comment:
I don't like usage of `let` for it. you can create separate `toSeparator` and `fromSeparator` variables